pkg/k8s/api.go

```diff
@@ -114,7 +114,7 @@ func (kubeapi *kubernetesApi) UrlFor(namespace string, extraPathStartingWithSlas
 	return generateKubernetesApiBaseUrlFor(kubeapi.apiSchemeHostAndPort, namespace, extraPathStartingWithSlash)
 }
 
-func NewK8sAPi(shell shell.Shell, k8sConfigFilesystemPathOverride string, apiHostAndPortOverride string) (KubernetesApi, error) {
+func NewK8sAPI(shell shell.Shell, k8sConfigFilesystemPathOverride string, apiHostAndPortOverride string) (KubernetesApi, error) {
```

Comment:
We should be able to remove the `apiHostAndPortOverride` argument to this method.